src/status_im2/contexts/onboarding/enable_notifications/view.cljs

```diff
@@ -0,0 +1,37 @@
+(ns status-im2.contexts.onboarding.enable-notifications.view
+  (:require [quo2.core :as quo]
+            [quo2.foundations.colors :as colors]
+            [react-native.core :as rn]
+            [status-im2.contexts.onboarding.enable-notifications.style :as style]
+            [utils.i18n :as i18n]
+            [status-im2.contexts.onboarding.common.background.view :as background]
+            [utils.re-frame :as rf]))
+
+(defn navigation-bar
+  []
+  [rn/view {:style style/navigation-bar}
+   [quo/page-nav
+    {:align-mid?  true
+     :mid-section {:type :text-only :main-text ""}
+    }]])
+
+(defn page
+  []
+  [rn/view {:style style/page-container}
+   [navigation-bar]
+   [rn/view {:style {:padding-horizontal 20}}
+    [quo/text
+     {:size   :heading-1
+      :weight :semi-bold
+      :style  {:color colors/white}} "Enable-notifications"]
```

Comment:
Localize string I suppose? Also is the hyphen correct?

Reply:
See my other comment on this :)

Reply:
Dismiss comment if the string will be replaced later